support/environment_test.go

```diff
@@ -0,0 +1,126 @@
+package support
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGetCodeFlareSDKVersion(t *testing.T) {
+	// Set the environment variable.
+	os.Setenv(CodeFlareTestSdkVersion, "1.2.3")
+
+	// Get the version.
+	version := GetCodeFlareSDKVersion()
+
+	// Assert that the version is correct.
+	if version != "1.2.3" {
+		t.Errorf("Expected version 1.2.3, but got %s", version)
+	}
```

Comment:
In the code above there is used Gomega for assertions.
Can you leverage it also here (and in other places) to keep consistency?

Reply:
Sure , will do